Package can't be used with NPM

# Steps to reproduce

1. Use a project that uses npm
2. Try to install the package

# Expected result

I can install the package from npm registry regardless of what package manager I use

# Actual result

```bash
$ npm i electron-typescript-ipc
npm ERR! code 1
npm ERR! path /Users/golergka/Projects/project-name/node_modules/electron-typescript-ipc
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
npm ERR! ╔═════════════════════════════════════════════════════════════╗
npm ERR! ║                                                             ║
npm ERR! ║   Use "pnpm install" for installation in this project.      ║
npm ERR! ║                                                             ║
npm ERR! ║   If you don't have pnpm, install it via "npm i -g pnpm".   ║
npm ERR! ║   For more details, go to https://pnpm.js.org/              ║
npm ERR! ║                                                             ║
npm ERR! ╚═════════════════════════════════════════════════════════════╝
```

Looks like I can only use this package with a project that uses pnpm.

--- a/package.json
+++ b/package.json
@@ -9,8 +9,7 @@
     "lint": "eslint src/**/*.ts",
     "prepublishOnly": "npm run build",
     "pack:local": "npm pack",
-    "link:local": "npm link",
-    "preinstall": "npx only-allow pnpm"
+    "link:local": "npm link"
   },
   "repository": {
     "type": "git",

